### PR TITLE
Codefresh Cron Triggers

### PR DESCRIPTION
**Pull request by osterman**
## what
* Document how to create scheduled pipelines with cron trigger

## why
* Useful for rebuilding datasets

### CODE REVIEW

File: content/release-engineering/codefresh/kubernetes-integration.md [hunk 1]
```diff
@@ -29,6 +32,10 @@ https://github.com/cloudposse/geodesic/blob/master/rootfs/usr/local/bin/codefres
 
 Enter the outputs of the script into the form and click save.
 
+{{% dialog type="info" icon="fa fa-info-circle" title="Note" %}}
+We recommend naming the configuration after the cluster name (e.g. `us-west-2-staging-example-org`).
```
Comment (goruha) [inline content/release-engineering/codefresh/kubernetes-integration.md]: `us-west-2-staging-example-com`

File: content/faq/how-to-perform-rollbacks.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ How to rollback application to previous version?
 
 # Answer
 
-[Helm]({{< relref "tools/helm.md" >}}) handles rollbacks out-of-the-box. Everytime a helm replease is performed, it creates a new revision. By pinpointing a specific helm revision and performing a rollback, the previous version will be redeployed. 
+[Helm]({{< relref "tools/helm.md" >}}) handles rollbacks out-of-the-box. Everytime a helm replease is performed, it creates a new revision. By pinpointing a specific helm revision and performing a rollback, the previous version will be redeployed.
```
Comment (aknysh) [inline content/faq/how-to-perform-rollbacks.md]: release

File: content/glossary/init-terraform.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+title: "init-terraform"
+description: 'The `init-terraform` script is a helper for configuring and then initializing terraform remote state incombination with the terraform-aws-tfstate-backend module.'
```
Comment (aknysh) [inline content/glossary/init-terraform.md]: in combination

File: content/glossary/init-terraform.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+terms:
+- init-terraform
+tags:
+- tfstate
+---
+
+{{% dialog type="info" icon="fa fa-book" title="Read More" %}}
+- This [`init-terraform`](https://github.com/cloudposse/geodesic/blob/master/rootfs/usr/local/bin/init-terraform) script is provided as part of geodesic.
+- The [terraform-aws-tfstate-backend]({{< relref "terraform-modules/supported/terraform-aws-tfstate-backend.md" >}}) module provides an encrypted S3 bucket for persisting state and a dynamodb for state locking.
```
Comment (aknysh) [inline content/glossary/init-terraform.md]: and a DynamoDB table for state locking

File: content/release-engineering/codefresh/cron-trigger.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+Codefresh supports [triggers](https://codefresh.io/docs/docs/pipeline-triggers/introduction-triggers/) which can execute a pipeline based on external webooks or cronjobs.
+
+# Configuration
+
+1. First setup a pipeline the way you would for any other kind of build.
+
+2. Navigate to the "Configuration" and select "Add Trigger"
+{{< img src="/assets/cron-trigger-7ac0cd4f.png" title="Pipeline Configuration" >}}
+
+3. Select "CRON" as the type and select "Next"
```
Comment (aknysh) [inline content/release-engineering/codefresh/cron-trigger.md]: click "Next"

**Pull request merged**